src/StageConfig.cc

```diff
@@ -157,24 +160,25 @@ void StageConfig::MachinesNeededPerStage(double tolerance) {
   // centrifuge feed flow (centrifuge.feed).
 
   // "Uranium Enrichment By Gas Centrifuge" D.G. Avery & E. Davies pg. 18
-  double cfeed_flow = (2 * DU_/centrifuge.M) * ((1 - cut_) / cut_) / pow((alpha_ - 1.), 2.);
+  double cfeed_flow =
+      (2 * DU_ / centrifuge.M) * ((1 - cut_) / cut_) / pow((alpha_ - 1.), 2.);
```

Comment:
```suggestion
      2 * DU_ * ((1 - cut_) / cut_) / pow((alpha_ - 1.), 2.);
```

If we are not including molar mass in line 114 above, we probably shouldn't here, correct?